packages/interactor/src/interaction.ts

```diff
@@ -1,3 +1,5 @@
+export const isInteraction = Symbol('interaction')
```

Comment:
We want this to be a global symbol, not local to the module. Otherwise, if the symbol is eval'd twice (which can easily happen with the way `node_modules` works) then you'll have many copies of the symbol floating around

```suggestion
export const isInteraction = Symbol.for('interaction');
```

Reply:
Didn't know about that, thanks.